config/routes.rb

```diff
@@ -17,5 +18,5 @@
   post '/login', to: "sessions#create"
   delete '/logout', to: "sessions#destroy"
   get '/sign_up', to: "welcome#new"
-
-end
+  get '/deactivate_account', to: "contractors#deactivate_account"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.